frontend/src/views/Lobby.tsx

```diff
@@ -108,6 +135,14 @@ function LobbyPage() {
           }}
         />
       ) : null}
+      { shouldRedirectToGame ? (
+        <Redirect
```

Comment:
I tested refreshing right as I clicked start game, and it redirected all the other users but kept the host in the lobby page.

Obviously we talked about this earlier as a total edge case, so no action needed. I wanted to note though, that in the future, I think we may be able to fix this by setting a variable in the `RoomDto` as `active` or something, which indicates that the game is ongoing and any user still in the lobby somehow should be redirected to the game page.

Food for thought. 🥘

Reply:
I also agree on this point of adding an `active` or other variable to the Room/RoomDto objects, as I think it has several design advantages:

1. Fixes the refreshing bug as @zpChris mentioned
2. A user tries to join a room that's already started - having an `active` parameter will allow us to prevent them from joining
3. Users can have all the info they need just from subscribing to a single socket endpoint
4. It allows us to keep track of valuable room/game info (e.g. which rooms have started) in the backend

Reply:
@helenli8181 Thoughts?